### PR TITLE
Lps 145373 2 21

### PR DESCRIPTION
**Pull request by slnn**
/cc @dantewang @tinatian

### CODE REVIEW

File: modules/test/company-sample-data-generation-test/src/testIntegration/java/com/liferay/company/sample/data/generation/test/CompanySampleDataGenerationTest.java [hunk 1]
```diff
@@ -243,17 +243,28 @@ private void _addUsers(
 		}
 	}
 
+	private void _exportCompanyData(String hostName) throws Exception {
+		Path companyCSVFilePath = _outputDirPath.resolve(_COMPANY_CSV);
+
+		try (BufferedWriter companyBufferedWriter = new BufferedWriter(
```
Comment (dantewang) [inline modules/test/company-sample-data-generation-test/src/testIntegration/java/com/liferay/company/sample/data/generation/test/CompanySampleDataGenerationTest.java]: @slnn Although it's OK to use `FileWriter` here, actually the `Files` API supports:

```
Files.newBufferedWriter(companyCSVFilePath, StandardOpenOption.CREATE, StandardOpenOption.APPED);
```

or 

```
Files.newBufferedWriter(companyCSVFilePath, _openOptions);

...

private static final OpenOption[] _openOptions = {StandardOpenOption.CREATE, StandardOpenOption.APPEND);
```

Comment (slnn) [reply modules/test/company-sample-data-generation-test/src/testIntegration/java/com/liferay/company/sample/data/generation/test/CompanySampleDataGenerationTest.java]: OK, I see, do you want me to resend the pull to apply the Files API?

Comment (dantewang) [reply modules/test/company-sample-data-generation-test/src/testIntegration/java/com/liferay/company/sample/data/generation/test/CompanySampleDataGenerationTest.java]: @slnn I'm also SFing other parts, so I will modify it together.

**Comment by dantewang** [comment]
See https://github.com/slnn/liferay-portal/pull/595, please apply, squash commits and resend.

**Comment by dantewang** [comment]
ci:close

**Pull request closed without merging**